Sources/AppStoreConnectCLI/Commands/TestFlight/BetaTesters/ListBetaTestersCommand.swift

```diff
@@ -0,0 +1,97 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct ListBetaTestersCommand: CommonParsableCommand {
+    static var configuration = CommandConfiguration(
+        commandName: "list",
+        abstract: "List beta testers")
+
+    @OptionGroup()
+    var common: CommonOptions
+
+    @Option(help: "The bundle ID of an application. (eg. com.example.app)")
+    var bundleId: String?
+
+    @Option(help: "The name of the beta group")
+    var betaGroupName: String?
+
+    @Option(help: "The ID of one build of an application")
+    var buildId: String?
```

Comment:
```suggestion
    var filterBuildId: String?
```

Reply:
Also, you should be able to specify these filter options in combinations.  The code below seems to assume you only use one at a time.